### PR TITLE
fix wrong review modal when logging in to rate when already rated (bug 1127347)

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: src/media/js/reviews.js [hunk 1]
```diff
@@ -80,42 +80,40 @@ define('reviews',
...
-                open_rating = false;
-                var $reviewButton = $('.review-button');
-                if (caps.widescreen()) {
-                    initReviewModal();
-                } else {
-                    z.page.trigger('navigate', $reviewButton.attr('href'));
-                }
+            var reviewButton = document.querySelector('.review-button');
+            if (caps.widescreen()) {
+                addReview.apply(reviewButton);
```
Comment (spasovski) [inline src/media/js/reviews.js]: So were weren't handling a widescreen flow for this before?

**Comment by spasovski** [comment]
Unrelated to this PR but I wonder if we want the edit review buttons to be 50% width each.



r+

File: src/media/js/reviews.js (hunk 1 above)
Comment (ngokevin) [inline src/media/js/reviews.js]: no we were if you check the deletions

**Comment by ngokevin** [comment]
there's a bug for that

**Comment by ngokevin** [comment]
ty

**Pull request merged**